xapian-core/include/xapian/weight.h

```diff
@@ -590,6 +590,101 @@ class XAPIAN_VISIBILITY_DEFAULT BM25Weight : public Weight {
     double get_maxextra() const;
 };
 
+/// Xapian::Weight subclass implementing the BM25+ probabilistic formula.
+class XAPIAN_VISIBILITY_DEFAULT BM25PlusWeight : public BM25Weight {
+
+  public:
+    /** Construct a BM25PlusWeight.
+     *
+     *  @param delta  A parameter for pseudo tf value to control the scale
+     *		      of the tf lower bound. Delta(δ) can be tuned for e.g
+     *		      from 0.0 to 1.5 but BM25+ can still work effectively
```

Comment:
"e.g." is from the Latin "exempli gratia" and means "for example", so "for e.g." would be "for for example"...